src/main/java/Controller/userController.java

```diff
@@ -1,4 +1,37 @@
 package Controller;
 
+import InputBoundary.loginInputBoundary;
+import InputData.loginInputData;
+
+
+/**
+ * Controller for login and register use cases.
+ */
 public class userController {
+    final loginInputBoundary loginInteractor, registerInteractor;
+    /**
+     * Constructor.
+     * @param loginInteractor use case interactor for adding a song to the space
+     * @param registerInteractor use case interactor for playing the space
+     */
+    public userController(loginInputBoundary loginInteractor, loginInputBoundary registerInteractor) {
+        this.loginInteractor = loginInteractor;
+        this.registerInteractor = registerInteractor;
+    }
+
+    /**
+     * function calling the use case for playing the space
+     * @param loginID input data for playing the space
+     */
+    public void login(loginInputData loginID){
```

Comment:
This inputData should be made in the controller rather than being passed in. So the controller class should have an inputData instance variable that is initialised in the constructor. You can then pass in the instance variable into your method